roles/variables/defaults/main.yml

```diff
@@ -416,7 +437,115 @@ kafka_controller_ssl_enabled: "{{ssl_enabled}}"
+kafka_controller_jolokia_active_controller_url: "{{ 'https' if kafka_controller_jolokia_ssl_enabled|bool else 'http' }}://{{ hostvars[inventory_hostname]|confluent.platform.resolve_hostname }}:{{kafka_controller_jolokia_port}}/jolokia/read/kafka.controller:type=KafkaController,name=ActiveControllerCount"
+kafka_controller_erp_clusters_url: "{{mds_http_protocol}}://{{ mds_advertised_listener_hostname | default(hostvars[inventory_hostname]|confluent.platform.resolve_hostname, True) }}:{{mds_port}}/kafka/v3/clusters"
+
+### Boolean to enable Prometheus Exporter Agent installation and configuration on kafka
+kafka_controller_jmxexporter_enabled: "{{jmxexporter_enabled}}"
+
+### Port to expose prometheus metrics. Beware of port collisions if colocating components on same host
+kafka_controller_jmxexporter_port: 8079
+
+### Path on Ansible Controller for Kafka Broker jmx config file. Only necessary to set for custom config.
+kafka_controller_jmxexporter_config_source_path: kafka.yml.j2
+
+### Destination path for Kafka Broker jmx config file
+kafka_controller_jmxexporter_config_path: /opt/prometheus/kafka.yml
+
+### Use to copy files from control node to kafka hosts. Set to list of dictionaries with keys: source_path (full path of file on control node) and destination_path (full path to copy file to). Optionally specify directory_mode (default: '0750') and file_mode (default: '0640') to set directory and file permissions.
+kafka_controller_copy_files: []
+
+### Replication Factor for internal topics. Defaults to the minimum of the number of brokers and can be overridden via default replication factor (see default_internal_replication_factor).
+kafka_controller_default_internal_replication_factor: "{{ [ groups['kafka_controller'] | default(['localhost']) | length, default_internal_replication_factor ] | min }}"
+
+### Boolean to enable the kafka's metrics reporter. Defaults to true if Control Center in inventory. Enable if you wish to have metrics reported to a centralized monitoring cluster.
+kafka_controller_metrics_reporter_enabled: "{{ confluent_server_enabled and 'control_center' in groups }}"
+
+### Use to set custom kafka properties. This variable is a dictionary. Put values true/false in quotation marks to perserve case.
+kafka_controller_custom_properties: {}
+
+### Use to add custom properties to variable kafka_controller_client_properties. This variable is a dictionary. Put values true/false in quotation marks to perserve case.
+kafka_controller_custom_client_properties: {}
+
+### Boolean to enable the embedded rest proxy within Kafka. NOTE- Embedded Rest Proxy must be enabled if RBAC is enabled and Confluent Server must be enabled
+kafka_controller_rest_proxy_enabled: false
```

Comment:
I believe rest_proxy might not work with the controller. Let's give it a try with the value true and see whether the server fails or not.

Reply:
yes, it doesn't work

Reply:
As per Ron's response, lets remove these props then.